compiler/src/dotty/tools/dotc/config/ScalaSettings.scala

```diff
@@ -163,13 +163,29 @@ private sealed trait WarningSettings:
     name = "-Wunused",
     helpArg = "warning",
     descr = "Enable or disable specific `unused` warnings",
-    choices = List("nowarn", "all"),
+    choices = List("nowarn", "all", "imports", "locals", "privates", "patvars", "explicits", "implicits", "params", "linted"),
```

Comment:
Does Scala 3 have a way to provide better help text? In Scala 2:

```
$> scalac -W
...
Warnings options:
...
  -Wunused:imports               Warn if an import selector is not referenced.
...
  -Wunused:implicits             Warn if an implicit parameter is unused.
...
  -Wunused:linted                -Xlint:unused.
...
```

In Scala 3:

```
-Wunused  Enable or disable specific `unused` warnings
          Choices nowarn, all, imports, locals, privates, patvars, explicits,
          implicits, params, linted
```

Reply:
I can try to improve this, when the main logic of the PR is done.
I had this to the _tasks_ at the top of the PR

Reply:
@lrytz This has been added in last commit, with Scala 2 help message